index.html

```diff
@@ -4669,105 +4714,130 @@ <h3>Data First Approaches</h3>
     <h2>Internationalization Considerations</h2>
 
     <p>
-There are a number of internationalization considerations implementers should be
-aware of when publishing data described in this specification. As with any web
-standards or protocols implementation, ignoring internationalization makes it
-difficult for data to be produced and consumed across a disparate set of
-languages and societies, which would limit the applicability of the
-specification and significantly diminish its value as a standard.
+There are a number of internationalization considerations implementers
+are advised to be aware of when publishing data described in this specification.
+As with any web standards or protocols implementation, ignoring
+internationalization makes it difficult for data to be produced and consumed
+across a disparate set of languages and societies, which would limit the
+applicability of the specification and significantly diminish its value as
+a standard.
+    </p>
+
+    <p>
+Implementers are strongly advised to read the Strings on the Web: Language and
+Direction Metadata document [[STRING-META]] published by the W3C
+Internationalization Activity as it elaborates upon the need to provide
+reliable metadata about text in order to support internationalization.
+Implementers are also urged to read the Implementation Guidance
+[[VC-IMP-GUIDE]] for the latest information related to this topic.
     </p>
 
     <p>
 This section outlines general internationalization considerations to take into
-account when utilizing this data model.
+account when utilizing this data model and is intended to highlight specific
+parts of the Strings on the Web: Language and Direction Metadata document
```

Comment:
`the Strings on the Web: Language and Direction Metadata document`
->
`the <em>Strings on the Web: Language and Direction Metadata</em> document`

Reply:
Fixed in f0b4f5063bad9633a8d0c5407fac3a2408ea4121.